drivers/gles2/rasterizer_scene_gles2.cpp

```diff
@@ -4129,9 +4129,9 @@ void RasterizerSceneGLES2::initialize() {
 }
 
 void RasterizerSceneGLES2::iteration() {
-	shadow_filter_mode = ShadowFilterMode(int(GLOBAL_GET("rendering/quality/shadows/filter_mode")));
+	shadow_filter_mode = ShadowFilterMode(int(GLOBAL_GET_CACHED(int32_t, "rendering/quality/shadows/filter_mode")));
 
-	const int directional_shadow_size_new = next_power_of_2(int(GLOBAL_GET("rendering/quality/directional_shadow/size")));
+	const int directional_shadow_size_new = next_power_of_2(GLOBAL_GET_CACHED(int32_t, "rendering/quality/directional_shadow/size"));
```

Comment:
Just noticing one or two types aren't lining up exactly, like here. Probably unimportant, but I thought I'd comment it.

Reply:
It's fine, it was deliberate.
Generally we're trying to move the code towards using fixed width types over the old style types, so any code that gets touched I'm trying to move over where applicable.